src/pages/Accounts/AccountToken/AccountToken.tsx

```diff
@@ -1,17 +1,16 @@
-import { Flex } from '@chakra-ui/react'
 import { CAIP19 } from '@shapeshiftoss/caip'
 import { useSelector } from 'react-redux'
 import { Redirect, useParams } from 'react-router-dom'
+import { Route } from 'Routes/helpers'
 import { AssetAccountDetails } from 'components/AssetAccountDetails'
-import { Page } from 'components/Layout/Page'
 import { AccountSpecifier } from 'state/slices/portfolioSlice/portfolioSlice'
 import { selectPortfolioAccountIdsSortedFiat } from 'state/slices/selectors'
 export type MatchParams = {
   accountId: AccountSpecifier
   assetId: CAIP19
 }
 
-export const AccountToken = () => {
+export const AccountToken = ({ route }: { route: Route }) => {
```

Comment:
Non-blocking, it's usually nice-to-have props as their own Type, either as an explicit props typing or a generic of `React.FC<T>`